pkg/run/interfaces.go

```diff
@@ -0,0 +1,24 @@
+package run
```

Comment:
We get cyclic dependencies otherwise.
The mocks are in a different package.
They import the original code and are imported by the tests in the same package.